CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.26.0 Release
+- Release Date: December 16, 2019
+- Server Versions Supported: Server v5.9+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Highlights
+
+#### ID-loaded push notifications (E20)
```

Comment:
this needs a bit more explanation of what it is

Reply:
Not sure about the name, if you read this as a plain mortal, do you understand what it means without reading the description? I mean the name is OK for internal use, but definitely not for others

Reply:
even in the system console is not shown with that name